src/pages/Product/Product.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Category } from "../../components/Filter/Category";
+import { Products } from "../../components/Products/Products";
+import { Navbar } from "../../components/Navbar/Navbar";
+import "./product.css";
+import { ProductData } from "../../components/Products/ProductData";
+import { Footer } from "../../components/Footer/Footer";
```

Comment:
You could do the same thing as mentioned above to make an index.js in components folder and then export all these from that and import together here.

e.g. 
```jsx
import {Category, Products, Navbar, ProductData, Footer} from "../../components";
```